app/assets/stylesheets/_variables.scss

```diff
@@ -46,9 +46,9 @@ $search-result-banner: 1.8em;
 }
 
 @mixin border-radius-small {
-  border-radius: 5px;
+  border-radius: .25rem;
 }
 
 @mixin border-radius-large {
-  border-radius: 25px;
```

Comment:
The reasoning was that I wanted to use rounded rem units so the browser does the calc. Since it was one pixel off, I wanted to use the mixins so it updates it site-wide for consistency, but I'm fine with switching it back.

Reply:
hmm I think thats some good reasoning and I'm sort of in favour of keeping the change now.
The only other issue I have with it though is that this file will now contain a mixture of em px and rem units which might be a little bit confusing from a standards point of view.